apps/frontend-pwa/src/pages/course/[courseId]/index.tsx

```diff
@@ -88,20 +94,23 @@ function CourseOverview({ courseId }: any) {
               />
             ))}
 
-            <Tabs.Tab
-              key="create"
-              value="create"
-              label="Gruppe erstellen/beitreten"
-            />
+            {(data.participantGroups?.length ?? 0) < 1 && (
+              <Tabs.Tab
+                key="create"
+                value="create"
+                label="Gruppe erstellen/beitreten"
+              />
+            )}
           </Tabs.TabList>
 
-          <Tabs.TabContent key="course" value="global" className="md:px-4">
-            {course.description && (
-              <div className="p-4 mb-4 border rounded bg-slate-100">
-                <Markdown content={course.description} />
-              </div>
-            )}
+          {course?.description && (
```

Comment:
For some reason, this tab is also shown, when the description is "null" in the database:
Additionally, we would have to consider the case where the user started editing the field as it might contain only break and newline characters, where this test would fail as well...
(image for description null)
<img width="1366" alt="Screenshot 2023-02-19 at 12 31 19" src="https://user-images.githubusercontent.com/80708107/219945257-29f48acd-f0d4-486a-8bef-982a2e9d1f1c.png">